docs/pages/federating-your-data/generating-tokens.mdx

```diff
@@ -0,0 +1,26 @@
+# Generating Tokens for Accessing Protected Objects
+
+To access a protected object in the Pelican data federation, the client must present a token signed by the Origin where the object is hosted on. The token then is used by Pelican servers to verify the object access permissions granted to the user, alone with other security checks to ensure the token is valid and not tempered by malicious attackers. Pelican takes advantage of JSON Web Token (JWT) to embed permission, user and server identity to the token.
+
+To allow user access the protected objects, the token must be generated by the Pelican Origin. Pelican Origin can be configured to use a third-party OIDC token issuer, in which case the client will be prompted to get a token from the issuer. By default, however, the Pelican Origin needs to generate the token by its built-in issuer and pass it to the client. This page documents different ways to generate the token to access the protected objects.
+
+## Pelican CLI
+
+The Pelican binary comes with a command `pelican origin token create` to generate the token. To generate a valid token, this command **MUST** be run on the same server where the Pelican Origin hosting the target object runs. On the Origin server, run:
+
+```bash copy
+pelican origin token create \
+    --scope "storage.read:/" \
+    --issuer https://localhost:8443 \
+    --claim "wlcg.ver=1.0" \
+    --subject origin \
+    --audience https://wlcg.cern.ch/jwt/v1/any
+```
+
+where:
+
+- `--scope` defines the access permission of the token. It follows [WLCG claim semantics](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/blob/master/profile.md#capability-based-authorization-scope). In this example, the token has `read` permission to the `/` prefix, meaning it can read any objects on this Origin. For other scope options, refer to [WLCG GitHub website](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/blob/master/profile.md#capability-based-authorization-scope).
+- `--issuer` tells Pelican the issuer URL it should embed in the token, which will be used by other Pelican servers to verify the integrity of the token. The issuer should be the publicly accessible URL of the Origin server.
+- `--claim` encodes an additional JWT claim, `wlcg.ver` with value `1.0` to the token. `wlcg.ver` is a special claim to indicate the version of the token profile. Pelican Origin accepts both `SciToken` token profile and `WLCG` token profile. Here we set the token profile to `WLCG`. See details [here](https://github.com/WLCG-AuthZ-WG/common-jwt-profile/blob/master/profile.md#common-claims).
+- `--subject` encodes the entity that issues the token.
+- `--audience` encodes the entities who requested the token. In most of cases, Pelican uses the special `any` audience.
```

Comment:
Do you think there's a need to talk about the scitokens profile this supports? I'm unsure, but it could be that some legacy origins only accept scitokens. Might be worth double checking.

Reply:
I'm not sure about the compatibility situations for all the origins but I think this should be a good starting point for most of the users